docs/cuproj/source/user_guide/cuproj_api_examples.ipynb

```diff
@@ -0,0 +1,192 @@
+    "cuProj is a GPU-accelerated Python library for cartographic coordinate projection and coordinate transformations between coordinate reference systems (CRS). The cuProj Python API provides an accessible interface to high-performance projections accelerated by CUDA-enabled GPUs. The API closely follows the [PyProj](https://pyproj4.github.io/pyproj/stable/) API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "caadf3ca-be3c-4523-877c-4c35dd25093a",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "This guide provides a working example for all of the python API components of cuProj.  \n",
+    "The following list links to each subsection.\n",
+    "\n",
+    "* [Installing cuProj](#Installing-cuProj)\n",
+    "* [Transformations with Transformer](#Transformations-with-Transformer)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "115c8382-f83f-476f-9a26-a64a45b3a8da",
+   "metadata": {},
+   "source": [
+    "## Installing cuProj\n",
+    "Read the [RAPIDS Quickstart Guide](  https://rapids.ai/start.html     ) to learn more about installing all RAPIDS libraries, including cuProj.\n",
+    "\n",
+    "If you are working on a system with a CUDA-enabled GPU and have CUDA installed, uncomment the  \n",
+    "following cell and install cuSpatial:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7265f9d2-9203-4da2-bbb2-b35c7f933641",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !conda create -n rapids-23.08 -c rapidsai -c conda-forge -c nvidia \\ \n",
+    "#     cuproj=23.08 python=3.10 cudatoolkit=12.0"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "051b6e68-9ffd-473a-89e2-313fe1c59d18",
+   "metadata": {},
+   "source": [
+    "For other options to create a RAPIDS environment, such as docker or build from source, see  \n",
+    "[RAPIDS Release Selector](  https://rapids.ai/start.html#get-rapids). \n",
+    "\n",
+    "If you wish to contribute to cuProj, you should create a source build using the included\n",
+    "devcontainer. Simply clone the github repository and open the folder in VSCode. VSCode will prompt\n",
+    "you to open the folder in a devcontainer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfb1e810",
+   "metadata": {},
+   "source": [
+    "## Transformations with Transformer\n",
+    "\n",
+    "The primary class in cuProj is the `Transformer` class, which is used to transform coordinates from one CRS to another. The `Transformer` class is created from a source CRS and a destination CRS, which can be specified using a CRS string, an EPSG code, or an `(<authority>, code)` tuple. The `Transformer` class can then be used to transform coordinates from the source CRS to the destination CRS.\n",
+    "\n",
+    "Currently only the EPSG authority is supported, and only a subset of the EPSG codes are supported. The following EPSG codes are supported:\n",
```

Comment:
>"Currently only the EPSG authority is supported"

Does this mean only the authority code method above is supported? If yes imo we should remove mention of the others and switch the wording to something like

>The `Transformer` class is created from a source CRS and a destination CRS, which currently can only be specified using an `(<authority>, code)` tuple.

Reply:
That's not the only way it can be specified. These are all valid:

```
4326 # EPSG integer
"4326" # EPSG string
"EPSG:4326" # "<authority>:code" string, as long as authority is "EPSG"
("EPSG", 4326) # (<authority>, code) tuple, as long as authority is "EPSG" 
```

Reply:
pytests cover all combinations.